evm_loader/program/src/evm/mod.rs

```diff
@@ -405,6 +419,170 @@ impl<B: Database> Machine<B> {
         Ok((status, step))
     }
 
+    #[allow(clippy::too_many_lines)]
+    #[maybe_async]
+    async fn execute_opcode(&mut self, backend: &mut B, opcode: u8) -> Result<Action> {
+        match opcode {
+            0x00 => self.opcode_stop(backend),
```

Comment:
Please revert opcodes back to lookup table.
Your changes are negatively impact performance.

Reply:
I also rely on this table in my PR, I've added opcode mnemonics to it.

Reply:
@anton-lisanin Done.